src/core/public/doc_links/doc_links_service.ts

```diff
@@ -38,8 +38,8 @@ export class DocLinksService {
       links: {
         dashboard: {
           drilldowns: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/drilldowns.html`,
-          drilldownsTriggerPicker: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/url-drilldown.html#trigger-picker`,
-          urlDrilldownTemplateSyntax: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/url-drilldown.html#templating`,
+          drilldownsTriggerPicker: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/supported-drilldowns.html`,
```

Comment:
This doesn't seem to go anywhere? 
I've tried: 
https://kibana_82081.docs-preview.app.elstc.co/guide/en/kibana/master/supported-drilldowns.html

Reply:
Mistake on my part. This should be guide/en/kibana/master/drilldowns.html#supported-drilldowns. 

I updated it, but I don't normally work in the .ts files. So if there is anything else I need to do, can you let me know?